gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,10 +1,6 @@
-#
-# Copyright 2017 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License").  See License in the project root for license information.
-#
-
-#Fri Apr 22 16:57:09 PDT 2016
+#Mon Feb 13 17:24:15 PST 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.2-bin.zip
```

Comment:
the switch from *-bin to *-all is meant to enable auto-complete in IDEA for gradle build files. the artifact is larger, but its a one-time setup cost and I think the end result is worth it